### PR TITLE
BGDIINF_SB-2128: Improvement of logging

### PR DESCRIPTION
**Pull request by IsabelleBzr**

### CODE REVIEW

File: app/helpers/validation_search.py [hunk 1]
```diff
@@ -203,6 +217,7 @@ def srid(self, value):
         if value in map(str, SUPPORTED_OUTPUT_SRS):
             self._srid = int(value)
         elif value is not None:
+            logger.warning("Unsupported spatial reference %s", value)
```
Comment (rebert) [inline app/helpers/validation_search.py:220]: In my log I'm getting the message twice (as warning and as error - I guess from the raised BadRequst):
`2022-02-17 13:29:14,894] DEBUG    - app.routes                 : GET /rest/services/ech/SearchServer
[2022-02-17 13:29:14,895] DEBUG    - flask_caching.backends.simplecache : get key 'get_topics_from_db' -> hit 
[2022-02-17 13:29:14,896] WARNING  - app.helpers.validation_search : Unsupported spatial reference 200
[2022-02-17 13:29:14,896] ERROR    - app                        : 400 Bad Request: Unsupported spatial reference 200
[2022-02-17 13:29:14,897] INFO     - app.routes                 : GET /rest/services/ech/SearchServer - 400 BAD REQUEST
[2022-02-17 13:29:14,898] INFO     - werkzeug                   : 127.0.0.1 - - [17/Feb/2022 13:29:14] "GET /rest/services/ech/SearchServer?sr=200&searchText=&lang=xx&type=locations&limit=2 HTTP/1.1" 400 -
`

Comment (rebert) [reply app/helpers/validation_search.py:220]: I guess it will be the same for the others with BadRequest

**Review by rebert — COMMENTED** [review]
@IsabelleBzr Thanks a lot.
I guess that we have duplicate log messages wherever a BadRequest or a InternalServerError is being been raised as well.

**Comment by ltshb** [comment]
@reber @IsabelleBzr  also the logs are then duplicated, I think it helps because the duplicated logs provide information of where the error happened. In theory the log from the `def handle_exception(err):` could be removed to avoid duplicated, but I would leave it as it is a general way of logging all errors even if we forget to log them in the code or if we forget to catch exception. What I would do though is changing all the logs to have the same severity, so changing the BadRequest logs to be also ERROR instead of WARNING.

**Comment by ltshb** [comment]
@IsabelleBzr I've added some logs, see commits above

**Review by ltshb — APPROVED** [review]
👍🏼

**Pull request merged**